bigquery/src/main/java/com/google/cloud/hadoop/io/bigquery/output/BigQueryOutputConfiguration.java

```diff
@@ -252,24 +252,49 @@ public static boolean getCleanupTemporaryDataFlag(Configuration conf) {
   }
 
   /**
-   * Gets the project id based on the given configuration. If the {@link
-   * BigQueryConfiguration#OUTPUT_PROJECT_ID_KEY} is missing, this resolves to referencing the
-   * {@link BigQueryConfiguration#PROJECT_ID_KEY} key.
+   * Gets the output dataset project id based on the given configuration.
+   *
+   * If the {@link BigQueryConfiguration#OUTPUT_PROJECT_ID_KEY} is missing, this
+   * resolves to referencing the {@link BigQueryConfiguration#PROJECT_ID_KEY} key.
    *
    * @param conf the configuration to reference the keys from.
    * @return the project id based on the given configuration.
    * @throws IOException if a required key is missing.
    */
-  public static String getProjectId(Configuration conf) throws IOException {
+  public static String getOutputProjectId(Configuration conf) throws IOException {
```

Comment:
This is a public method, to keep backwards compatible, I don't think we should rename it. Also, in the context of BigQueryOutputConfiguration, getProjectId is better than getOutputProjectId. But we can add comments to explain the difference between the 2 project IDs.